NuRadioReco/modules/io/rno_g/readRNOGDataMattak.py

```diff
@@ -0,0 +1,644 @@
+
+
+class readRNOGData:
+
+   def begin(self, 
+             data_dirs,  
+             read_calibrated_data=False,
+             select_triggers=None,
+             select_runs=True,
+             apply_baseline_correction=True,
+             convert_to_voltage=True,
+             selectors=None,
+             run_table_path=None,
+             run_types=["physics"],
+             run_time_range=None,
+             max_trigger_rate=1 * units.Hz,
+             mattak_backend="auto",
+             log_level=logging.INFO):
+      """
+
+      Parameters
+      ----------
+
+      data_dirs: list of strings / string
+         Path to run directories (i.e. ".../stationXX/runXXX/")
+         
+      read_calibrated_data: bool
+         If True, read calibrated waveforms from Mattak.Dataset. If False, read "raw" ADC traces.
+         (temp. Default: False)
+         
+      select_triggers: str or list(str)
+         Names of triggers which should be selected. Convinence interface instead of passing a selector
+         (see "selectors" below. (Default: None) 
+         
+      select_runs: bool
+         If True, use information in run_table to select runs (based on run_type, run_time, trigger_rate, ...).
+         If the run_table is not available no selection is performed (and the programm is not interrupted, 
+         only an error message is raised). See parameters to configure run selection. (Default: True)
+         
+      Other Parameters
+      ----------------
+      
+      apply_baseline_correction: bool
+         Only applies when non-calibrated data are read. If true, correct for DC offset.
+         (Default: True)
+
+      convert_to_voltage: bool
+         Only applies when non-calibrated data are read. If true, convert ADC to voltage.
+         (Default: True)
+
+      selectors: list of lambdas
+         List of lambda(eventInfo) -> bool to pass to mattak.Dataset.iterate to select events.
+         Example: trigger_selector = lambda eventInfo: eventInfo.triggerType == "FORCE"
+         
+      run_table_path: str
+         Path to a run_table.cvs file. If None, the run table is queried from the DB. (Default: None)
+         
+      run_types: list
+         Used to select/reject runs from information in the RNO-G RunTable. List of run_types to be used. (Default: ['physics'])
+         
+      run_time_range: tuple
+         Specify a time range to select runs (it is sufficient that runs cover the time range partially). 
+         Each value of the tuple has to be in a format which astropy.time.Time understands. A value can be None 
+         which means that the lower or upper bound is unconstrained. If run_time_range is None no time selection is
+         applied. (Default: None)
+         
+      max_trigger_rate: float
+         Used to select/reject runs from information in the RNO-G RunTable. Maximum allowed trigger rate (per run) in Hz.
+         If 0, no cut is applied. (Default: 1 Hz)
```

Comment:
It seems I'm only leaving pedantic comments today, but you can of course ignore them. Here, wouldn't the default 0 (no cut) make more sense? E.g. say I want to look at a specific couple of runs (maybe I'm interested in wind events...), then some of them will by default be rejected, which might be easy to miss and tricky to troubleshoot (though I see at least a logger message is emitted at the default log level).

Reply:
fine with me